modules/kubernetes/linkerd/charts/linkerd-extras/templates/linkerd-identity-issuer.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: linkerd-identity-issuer
 spec:
   secretName: linkerd-identity-issuer
+  revisionHistoryLimit: 3
```

Comment:
Perhaps we should have the linkerd certs a bit higher? Saving it for 4 days at least? (Something happens on Friday, we look at it on Monday)